app/views/auth/login.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Log In</h1>
+<% if @errors %>
```

Comment:
this could be extracted out and rendered as a partial since you are reusing it and/or will be reusing it